i18n/locales/en.ts

```diff
@@ -32,11 +32,9 @@ const translations = {
     project: {
       title: 'Built with fun',
       paragraph_one:
-        'Saving Satoshi is a light-hearted first point of contact for developers who are curious about bitcoin development. Saving Satoshi has a mix of technical text and code-based challenges, but the challenges should be doable for anyone with basic coding skills. The project is free and open source and all the art has been generated using text-to-image (aka AI) tools like Midjourney.',
+        'Saving Satoshi is a light-hearted first point of contact for coders who are curious about bitcoin development. Saving Satoshi has a mix of technical text and code-based challenges, but the challenges should be doable for anyone with basic coding skills. The project is free and open source and all the art has been generated using text-to-image tools like Midjourney.',
       paragraph_two:
-        'This version of the project is currently in a demo state with a goal of getting feedback on the overall experience and providing a proof of concept for early users. In light of this, user progress is stored locally in the browser and the story along with images are not finalized.',
-      paragraph_three:
```

Comment:
You removed paragraph_three which has this result:

![image](https://user-images.githubusercontent.com/10757768/233716095-8a764901-1ace-4590-8b3f-4d30d8521c6d.png)

You should also remove the following code from `app/[lang]/about/page.tsx` on line 52:

```tsx
<p>{t('about.project.paragraph_three')}</p>
```

Reply:
Good catch thanks! Changes made.